packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTask.spec.js

```diff
@@ -114,174 +116,158 @@ describe('SurveyTask', function () {
           expect(stripesFilterRemoveButton).to.be.null()
           
           await user.click(stripesFilterButton)
-          characteristicsSection = getByTestId('characteristics')
+          characteristicsSection = screen.getByTestId('characteristics')
           stripesFilterRemoveButton = within(characteristicsSection).getByTestId('remove-filter-PTTRN-STRPS')
           expect(stripesFilterRemoveButton).to.be.ok()
         })
 
         it('should show the choices that match the filter', async function () {
-          const user = userEvent.setup({ delay: null })
-          const { getAllByRole, getByRole, getByTestId } = render(<DefaultStory />)
-          const filterButton = getByRole('button', { name: 'SurveyTask.CharacteristicsFilter.filter' })
-          await user.click(filterButton)
-          const redFilterButton = getByTestId('filter-CLR-RD')
+          const redFilterButton = screen.getByTestId('filter-CLR-RD')
           expect(redFilterButton).to.be.ok()
 
           // the following user events filter choices by the color red and then close the characteristics filters, returning to the list of choices.
 
           await user.click(redFilterButton)
           await user.click(filterButton)
-          const choiceButtons = getAllByRole('menuitemcheckbox')
+          const choicesMenu = screen.getByTestId('choices-menu')
+          const choiceButtons = within(choicesMenu).getAllByRole('menuitemcheckbox')
           expect(choiceButtons.length).to.equal(3)
           expect(choiceButtons[0]).to.have.text('Aardvark')
           expect(choiceButtons[1]).to.have.text('Kudu')
           expect(choiceButtons[2]).to.have.text('Fire')
         })
 
         it('should persist filters after a choice is selected', async function () {
-          const user = userEvent.setup({ delay: null })
-          const { getAllByRole, getByRole, getByTestId } = render(<DefaultStory />)
-          const filterButton = getByRole('button', { name: 'SurveyTask.CharacteristicsFilter.filter' })
-
-          // open the characteristics filters
-          await user.click(filterButton)
-          const redFilterButton = getByTestId('filter-CLR-RD')
+          const redFilterButton = screen.getByTestId('filter-CLR-RD')
           // click/apply the red filter, which filters 6 choices to 3 choices
           await user.click(redFilterButton)
           // close the characteristics filters
           await user.click(filterButton)
-          const fireChoiceButton = getByRole('menuitemcheckbox', { name: 'Fire' })
-          expect(fireChoiceButton).to.be.ok()
-
+          const fireChoiceButton = screen.getByLabelText('Fire')
           // select the Fire choice
           await user.click(fireChoiceButton)
-          const identifyButton = getByRole('button', { name: 'SurveyTask.Choice.identify' })
-          expect(identifyButton).to.be.ok()
-
+          const identifyButton = screen.getByLabelText('SurveyTask.Choice.identify Fire')
```

Comment:
```suggestion
          const identifyButton = screen.getByLabelText('SurveyTask.Choice.identify')
```
Why has the choice been added to the button label?